### PR TITLE
Uxd 907 listbox keyboard

### PR DESCRIPTION
**Pull request by oscarkwan**
### Purpose 🚀
- listbox keyboard cypress tests

Screen Shot 2021-03-18 at 12 46 07 PM

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/UXD-907-listbox-keyboard

### CODE REVIEW

File: packages/ListBox/test/cypress/ListBox.a11y.cypress.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  cy.get(selectors.popoverList)
+    .findByText(optionName)
+    .should("have.focus");
+}
+
+describe("ListBox MultiSelect a11y", () => {
+  beforeEach(() => {
+    cy.visitStorybook(`${storyPrefix}-backyard-tests--a-11-y-tests-story`);
+    toggleDropdown();
+    cy.wait(350);
```
Comment (oscarkwan) [inline packages/ListBox/test/cypress/ListBox.a11y.cypress.js:25]: Wait's are needed throughout this test to make sure that the tests are not flaky. This is due to the focus timeouts / popover animation

Comment (nahumzs) [reply packages/ListBox/test/cypress/ListBox.a11y.cypress.js:25]: That popover animation :/ still not sure of the benefits of having it lol... like the animation in our panels...

File: packages/ListBox/test/cypress/ListBox.a11y.cypress.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  });
+
+  it("focus should be moved to the filter input once dropdown is opened", () => {
+    cy.get(selectors.filterInput).should("have.focus");
+  });
+
+  it("focus should be moved to the option once arrow key down is pressed", () => {
```
Comment (nahumzs) [inline packages/ListBox/test/cypress/ListBox.a11y.cypress.js:32]: `while on trigger`? focus should be move

File: packages/ListBox/test/cypress/ListBox.a11y.cypress.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  it("focus should be moved to the filter after going to first option and arrowing up", () => {
+    cy.get(selectors.filterInput).should("have.focus");
+    arrowDownKeyPress();
+    optionHasFocus("The Joker");
+    arrowUpKeyPress();
+    cy.get(selectors.filterInput).should("have.focus");
+  });
+
+  it("focus moves to first option in list after selecting options", () => {
```
Comment (nahumzs) [inline packages/ListBox/test/cypress/ListBox.a11y.cypress.js:53]: this legend is confusing you are testing the focus on the filter, but the description implies you are focusing on the first option. 

When is multi `should not` focus on any option if there are or not selected items in play after opening and clicking ↓

**Comment by nahumzs** [comment]
are we testing disabled options and rawItem to be no selected if the user press space or enter? or that would be part of another PR?

**Pull request closed without merging**